jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -117,6 +117,15 @@
                 export KUBE_AUTOSCALER_MAX_NODES=5
                 export KUBE_AUTOSCALER_ENABLE_SCALE_DOWN=true
                 export KUBE_ADMISSION_CONTROL="NamespaceLifecycle,InitialResources,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota"
+        - 'gce-alpha-features': # kubernetes-e2e-gce-alpha-features
+            description: 'Run alpha feature E2E tests on GCE.'
+            timeout: 180
+            job-env: |
+              export KUBE_FEATURE_GATES="AllAlpha=true"
```

Comment:
`AllAlpha`. That change was in https://github.com/kubernetes/kubernetes/pull/31140